l10n_it_payment_reason/hooks.py

```diff
@@ -0,0 +1,48 @@
+from openupgradelib import openupgrade
+
+
+def module_migration(cr):
```

Comment:
Potresti rinominarlo in `rename_old_italian_module`? Mi sembra più coerente con quello che fa.
Grazie!